app/assets/stylesheets/govuk-component/_taxonomy-sidebar.scss

```diff
@@ -19,29 +19,21 @@
     .related-content {
       margin: 0 0 0 15px;
       padding: 0;
-      list-style-type: none;
-      line-height: 1.5;
 
-      // We use a pseudo element to add bullet points to related links. This
-      // makes the bullets clickable and highlights them on link hover. With
-      // this approach, IE8 erroneously applies the anchor tag's underline to
-      // the pseudo element. Specifying separate text-decoration rules for the
-      // anchor tag and a nested span fixes this.
+      li {
+        font-size: 12px;
+      }
+
       a {
         display: block;
-        text-decoration: none;
+        font-size: 16px;
+        line-height: 1.5;
 
-        &:before {
-          content: "•";
-          display: inline-block;
-          width: 15px;
-          margin-left: -15px;
+        span {
```

Comment:
It's needed for vertical alignment of the bullet (which is slightly off because of its font size being different to that of the text) - I thought about it over the weekend and I think I want to slightly tweak this. Will change to WIP and add a comment in the CSS.